commands/files.rb

```diff
@@ -0,0 +1,42 @@
+require 'json'
+require_relative '../lib/const'
+require_relative '../lib/convert_size'
+
+class Command
+  def self.files(pkg)
+    # Check if the package is even installed first, as this is the most likely reason we cannot find a filelist.
+    device_json = JSON.load_file(File.join(CREW_CONFIG_PATH, 'device.json'))
+    if device_json['installed_packages'].none? { |entry| entry['name'] == pkg.name }
+      puts "Package #{pkg.name} is not installed.".lightred
+      return
+    end
```

Comment:
We use this conditional in several places.  What are your thoughts of creating a generic function something like `if package_installed(pkg.name)` to abstract this kludgy logic?

Reply:
Another method to determine if a package is installed is if the meta filelist exists.

Reply:
In fact, it might make sense to include both checks in case device.json is corrupted.  If you want to take it a step further, you could also do some kind of self-healing to fix a corrupted device.json if the filelist is missing.  I would do a `crew remove` of the package before trying to update device.json, however, in cases where the files of the package are still hanging around.

Reply:
I'm saying abstract the logic into a single function instead of including both checks here.

Reply:
We only use it in two places right now-- I don't think the savings are worth it. 

I'll keep an eye on it as I add more commands though, and abstract if it makes sense.

Reply:
It's not about savings so much as reuse.  We don't want to keep adding the same logic anytime we need to determine if a package is installed.  Is it too difficult to add?  I think @satmandu needed it for something recently.

Reply:
Fair enough-- that gives me an excuse for doing a `lib/package.rb` refactor that's been on the back of my mind, so I'll do that in another PR. I can update these files after I create and merge that PR, so don't feel the need to wait on it.